target/linux/ramips/dts/mt7621_cudy_wr1300-v3.dts

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621_cudy_wr1300-v2v3.dtsi"
+
+/ {
+	compatible = "cudy,wr1300-v3", "mediatek,mt7621-soc";
+	model = "Cudy WR1300 v3";
+};
+
+&pcie0 {
+	wifi@0,0 {
+		compatible = "mediatek,mt76";
+		reg = <0x0000 0 0 0 0>;
+		nvmem-cells = <&eeprom_factory_0>, <&macaddr_bdinfo_de00>;
+		nvmem-cell-names = "eeprom", "mac-address";
+		ieee80211-freq-limit = <2400000 2500000>;
```

Comment:
Add this node should fix the 2 GHz LED issue. As for 5 GHz LED, if we don't know the actual GPIO number, it can not be fixed.
```
		led {
			led-sources = <0>;
			led-active-low;
		};
```

Please confirm the fix.

Reply:
This works, I have included it in the patch.

Reply:
> if we don't know the actual GPIO number

Where can we find it in their factory rom?

What does their code do exactly:
enable: `iwpriv rai0 set led_setting=1-0-0-0-2-0-0-2`
disable: `iwpriv rai0 set led_setting=1-0-0-0-2-0-0-1`

Reply:
> What does their code do exactly:
> enable: `iwpriv rai0 set led_setting=1-0-0-0-2-0-0-2`
> disable: `iwpriv rai0 set led_setting=1-0-0-0-2-0-0-1`

If you are interested, you can track the source code of the mt7613.
https://github.com/hanwckf/rt-n56u/tree/master/trunk/proprietary/rt_wifi/rtpci/4.1.X.X

Reply:
> This works, I have included it in the patch.

Please insert one new line before `led {`.

Reply:
The LED pin should be '1' for MT7613 (WPS LED). Perhaps mt76 driver doesn't support mt7613 LED at all.

Reply:
Ok, let's do this some other time.